actionview/lib/action_view/layouts.rb

```diff
@@ -280,7 +280,11 @@ def layout(layout, conditions = {})
       def _write_layout_method # :nodoc:
         silence_redefinition_of_method(:_layout)
 
-        prefixes = /\blayouts/.match?(_implied_layout_name) ? [] : ["layouts"]
+        if _implied_layout_name.nil?
+          ["layouts"]
```

Comment:
This is not assigned to `prefixes`.
It seems better to keep the condition together:
```ruby
prefixes = if _implied_layout_name && /\blayouts/.match?(_implied_layout_name)
  []
else
  ["layouts"]
end
```